Subnets.tf

```diff
@@ -0,0 +1,110 @@
+
+resource "aws_subnet" "Geschäftsführung" {
+  vpc_id     = aws_vpc.opt-vpc.vpc_id
+  cidr_block = "10.0.0.0/8"
+  availability_zone = "us-west-2a"
+
+  tags = {
+    Name = "Geschäftsführung"
+  }
+}
+
+resource "aws_subnet" "Geschäftsführung-Ersatz" {
```

Comment:
see first comment and Ersatz sounds strange.

Maybe a and b for availability zone